### PR TITLE
Comment

### PR DESCRIPTION
**Pull request by caovanson06112001**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/51563)

## WHAT (optional)
- user can comment books and use action cable notifycation for admin

## HOW
- Add gem redis, make model comment

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

## Evidence (Screenshot or Video)
image

### CODE REVIEW

**Comment by caovanson06112001** [comment]
ready

File: app/channels/comment_channel.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class CommentChannel < ApplicationCable::Channel
+  def subscribed
+    stream_from "comment_channel"
+  end
+
+  def unsubscribed
+    # Any cleanup needed when channel is unsubscribed
+  end
```
Comment (anlvq-1058) [inline app/channels/comment_channel.rb:8]: bỏ thử đi xem đc ko em

File: app/views/admin/books/index.html.erb [hunk 2]
```diff
@@ -39,7 +39,7 @@
                   <div class="">
                     <h5>
                       <%= link_to t("books.add"), new_admin_book_path, class:"btn btn-primary" %>
-                      <%= link_to t("books.analytics"), "#", class:"btn btn-primary" %>
+                      <%# <%= link_to t("books.analytics"), "#", class:"btn btn-primary" %>
```
Comment (anlvq-1058) [inline app/views/admin/books/index.html.erb:42]: bỏ dòng này đi e

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by caovanson06112001** [comment]
ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**